scripts/build/golines.sh

```diff
@@ -42,9 +42,9 @@ else
 fi
 
 # Find all .go files in the project directory and its subdirectories, ignoring .pb.go and .pb_encoding.go files
-find "${ROOT_DIR}" -type f -name "*.go" ! -name "*.pb.go" ! -name "*.pb_encoding.go" | while read -r file; do
+find "${ROOT_DIR}" -type f -name "*.go" ! -name "*.pb.go" ! -name "*_test.go" ! -name "*.pb_encoding.go" | while read -r file; do
```

Comment:
q: why do we skip tests here?

Reply:
golines is formatting lines and comments under the specified line length. But certain tests have byte arrays or literal values that we don’t want to truncate for sake of formatting